packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.tsx

```diff
@@ -365,7 +366,11 @@ export class AtomicCommerceInterface
   }
 
   private initSearchStatus() {
-    this.searchStatus = buildSearch(this.engine!);
```

Comment:
@louis-bompart I'm not sure this change is right 🤔 Would love a second pair of eyes on it.

My understanding is that the commerce interface is used for search and listing. Its status should therefore reflect either the search or listing one, not only the search one.

Reply:
This change seems 💯 correct.

Reply:
I personally think this is fine and the previous behavior was wrong; otherwise I don't see how we're going to be able to support URL state in PLPs

Reply:
Now, since this interface is used possibly used for recs as well, should we type the urlManager to be conditionally present?

Reply:
I'll leave this as-is for now as a few atomic components use an if/else if with product-listing and search. Hence, if we do what I propose we'll need to apply it in a few places. I propose we revisit this later.